### PR TITLE
US95684 Add activity description rel and class

### PR DESCRIPTION
**Pull request by ryantmer**

### CODE REVIEW

File: d2l-hm-constants-behavior.html [hunk 1]
```diff
@@ -43,7 +43,8 @@
 				overdue: 'https://activities.api.brightspace.com/rels/overdue',
 				nextPeriod: 'https://activities.api.brightspace.com/rels/next-period',
 				previousPeriod: 'https://activities.api.brightspace.com/rels/previous-period',
-				feedback: 'https://activities.api.brightspace.com/rels/feedback'
+				feedback: 'https://activities.api.brightspace.com/rels/feedback',
+				description: 'https://activities.api.brightspace.com/rels/description'
```
Comment (CodeBaboon) [inline d2l-hm-constants-behavior.html]: Presumably activity types that don't have descriptions (like assignments) just don't have a link with this rel (or a class of the type)?

Comment (ryantmer) [reply d2l-hm-constants-behavior.html]: The current usage of `description` is even broader - just indicates a subentity/link that is text-based. In this case, text feedback entities will have a `description` class/rel (along with a `richtext` class if they've also got `properties.html` along with just the `properties.text`)

Comment (CodeBaboon) [reply d2l-hm-constants-behavior.html]: But that's what the `richtext` type is for, isn't it? Those have `text` and `html` properties.

Comment (ryantmer) [reply d2l-hm-constants-behavior.html]: Oh! Wasn't aware that that was the case, I thought `richtext` was only referring to the `html` part. In that case, this change is not required :D

**Pull request closed without merging**